alphagov/paas-cf
# [#169336585] Update AWS RDS certificate bundle

What
----
Updates the AWS RDS CA certificate bundle to a bundle that includes the new certificate for March 2020.

How to review
-------------
1. Compare the SHA sum of the bundle in source versus the one downloaded from Amazon
```
shasum -a 256 <(yq read manifests/cf-manifest/data/000-aws-rds-combined-ca-bundle-pem.yml  'aws_rds_combined_ca_bundle' | tr -s '\n')

shasum -a 256 <(curl -sL https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem)
```
2. Check the contents of the certificate bundle
```
openssl crl2pkcs7 -nocrl \
  -certfile <(yq read manifests/cf-manifest/data/000-aws-rds-combined-ca-bundle-pem.yml 'aws_rds_combined_ca_bundle') \
  | openssl pkcs7 -print_certs -text -noout \ 
  | grep -e 'Subject:' -e 'Not After'
```
3. Run it down your dev env

Who can review
--------------
Anyone
